R/incidence.R

```diff
@@ -187,26 +192,25 @@ incidence.numeric <- function(dates, interval = 1L, ...) {
 
 #' @export
 #' @rdname incidence
-#' @param iso_week A logical value indicating if dates of the weekly incidence
-#'   should be using ISO week. Only applies when \code{interval = 7}. Defaults
-#'   to be TRUE.
+#'
+#' @param iso (Only applicable to Date objects) When `TRUE` (default) and the
+#'   `interval` one of "week", "month", "quarter", or "year", then this will
+#'   cause the bins for the counts to start at the beginning of the interval
+#'   (See Note). This is overridden by defining a non-NULL `first_date`.
 
-incidence.Date <- function(dates, interval = 1L, iso_week = TRUE,
+incidence.Date <- function(dates, interval = 1L, iso = TRUE, first_date = NULL,
                            last_date = NULL, ...) {
   ## make sure input can be used
-  if (!is.logical(iso_week)) {
-    stop("The argument `iso_week` must be either `TRUE` or `FALSE`.")
-  }
-
-  if (!is.null(last_date) && !inherits(last_date, "Date")) {
-    stop("last_date is not a Date object")
+  if (!is.logical(iso)) {
+    stop("The argument `iso` must be either `TRUE` or `FALSE`.")
   }
   out <- make_incidence(dates = dates,
                         interval = interval,
+                        first_date = first_date,
                         last_date = last_date,
-                        iso_week = iso_week,
+                        iso = iso,
```

Comment:
As discussed, probably best renamed to `standard`, `common`, `canonical` ... (my pref is `standard`) and provide info in details:
- weeks = isoweeks
- month = 1st of each month
etc.

Reply:
I'd leave this to a separate PR / direct commit though

Reply:
I think it'll be better to resolve this in this PR than create a new one.

Reply:
This has been done as of 5ca529e